### DISCUSSION TITLE
Distinguishable playlists (Label of the music provider on playlist?)

### DISCUSSION THREAD

**Discussion by HA-TB303** [music-assistant/support#2496] [Feature requests and ideas]
When having configured multiple music providers it's quite hard to see which playlist belongs to who.

Situation:

- 3 person family
- Each family member uses his own YouTube music account
- All 3 accounts configured in Music Assistant:
![CleanShot 2024-06-17 at 13 02 09](https://github.com/music-assistant/hass-music-assistant/assets/9609805/2eee7c1c-f3cf-4b05-bde7-8fb4210481d5)

It then get's quite hard to see which playlist belongs to which person/account.

Possible solution: It could help if you could put a custom color/name label on a Music provider (or a label containing the music provider name) and this label would then overlay on the playlist like this:
![CleanShot 2024-06-17 at 13 06 24](https://github.com/music-assistant/hass-music-assistant/assets/9609805/9e3228c3-468f-4b8a-9f5e-93a420209163)

Yes, I could prefix the playlist name with the name of the person like: "Michel - Best of Queen"  but this seriously limits the space for the playlist name.

**Comment by OzGav** [comment]
This has been implemented

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
